pyaedt/modeler/cad/Modeler.py

```diff
@@ -1625,8 +1661,9 @@ def create_coordinate_system(
             Enumerator ``pyaedt.generic.constants.VIEW`` can be used.
 
             .. note::
-               Because the ``"rotate"`` option is obsolete, use
-               ``mode="axis"`` instead.
+              For back compatibility, ``mode="view"`` and ``view="rotate"`` is the same as
+              ``mode="axis"``. Because the "rotate" option in the "view" mode is obsolete, use
+              ``mode="axis"`` instead.
```

Comment:
```suggestion
              For backward compatibility, ``mode="view"`` and ``view="rotate"`` are the same as
              ``mode="axis"``. Because the "rotate" option in the "view" mode is obsolete, use
              ``mode="axis"`` instead.
```